### PR TITLE
[gbsyncd] Build docker-gbsyncd-broncos image

### PR DESCRIPTION
**Pull request by jimmyzhai**
#### Why I did it
The libsaibroncos debian package is published at `$(LIBSAI_BRONCOS)_URL`. Enable building docker-gbsyncd-broncos image on PLATFORM **broadcom**.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: platform/components/docker-gbsyncd-broncos.mk [hunk 1]
```diff
@@ -1,5 +1,8 @@
-LIBSAI_BRONCOS = libsaibroncos_0.0.1_amd64.deb
-$(LIBSAI_BRONCOS)_URL =
+LIBSAI_BRONCOS_VERSION = 3.8
+LIBSAI_BRONCOS_BRANCH_NAME = REL_3.8
+LIBSAI_BRONCOS_URL_PREFIX = "https://sonicstorage.blob.core.windows.net/public/sai/bcmpai/$(LIBSAIBCM_BRANCH_NAME)/$(LIBSAIBCM_XGS_VERSION)"
```
Comment (lguohan) [inline platform/components/docker-gbsyncd-broncos.mk:3]: should not be xgs version.

Comment (jimmyzhai) [reply platform/components/docker-gbsyncd-broncos.mk]: Thanks, fix this typo.

**Review by Blueve — APPROVED** [review]

**Comment by yxieca** [comment]
@jimmyzhai  this change causes build failure for 202205 branch if cherry-picked alone. Please raise separate PR or figure out the whole dependency chain.

**Pull request merged**